app/models/allocation.rb

```diff
@@ -54,6 +59,8 @@ def user_punches
       .decorate
   end
 
+
+
```

Comment:
extra space

Reply:
Removed on d610d771633ab737a460bfb7468c0a604431272b, thanks for the catch